lib/cc-reno.c

```diff
@@ -26,15 +26,19 @@
 #define QUICLY_MIN_CWND 2
 #define QUICLY_RENO_BETA 0.7
 
-void quicly_cc_init(quicly_cc_t *cc)
+void quicly_cc_init(quicly_cc_t *cc, uint32_t max_udp_payload_size)
 {
     memset(cc, 0, sizeof(quicly_cc_t));
-    cc->cwnd = QUICLY_INITIAL_WINDOW * QUICLY_MAX_PACKET_SIZE;
+    cc->cwnd = QUICLY_INITIAL_WINDOW * max_udp_payload_size;
```

Comment:
maybe rename this to QUICLY_INITIAL_WINDOW_PACKETS

Reply:
Please see my comment above.